doc/polyglossia.tex

```diff
@@ -44,14 +43,77 @@
 \def\Ie{\textit{I.e.,}\xspace}
 \def\etc{\@ifnextchar.{\textit{etc}}{\textit{etc.}\@\xspace}}
 
+%% Commands for documenting options
+\ExplSyntaxOn
+
+\NewDocumentCommand \pgoption {m}
+  {
+    \textcolor {pgblue} { \bfseries #1 }
+  }
+
+\NewDocumentCommand \pgdefault {m}
+  {
+    \textit{#1}
+  }
+
+\NewDocumentCommand \pgdefaultchoice {m}
+  {
+    \underline{#1}
+  }
+
+\NewDocumentCommand \pgchoicekey {omom}
+  {
+    \pgoption {#2}
+    \IfValueT {#3} { \footnote {#3} }
+    \IfValueT {#1} { \new {#1} }
+    \c_space_tl = \c_space_tl
+    \clist_set:Nn \l_tmpa_clist {#4}
+    \seq_set_from_clist:NN \l_tmpa_seq \l_tmpa_clist
+    \seq_use:Nnnn \l_tmpa_seq {~or~} {,~} {,~or~}
+    \par
+  }
+
+\NewDocumentCommand \pgboolkey {omo}
```

Comment:
It would be more convenient if we could have something like `\pkgoption[bool]` rather than all these separate commands.

Reply:
Well, I could merge the three commands concerning Boolean keys, but that would mean that a different delimiter would be needed for one optional argument. A command with two subsequent optional arguments is dangerous if one is left out by the user. I could define 
`\pgboolkey[<version>]{<key>}(<footnote>)[<default value>]`
using round brackets instead of square ones for the optional footnote.
Merging all commands concerning option descriptions would be too difficult as they need different types of arguments.

Reply:
Remember that these macros are not used by many users, but basically only by us. Why not use keyval syntax?

Reply:
It is not worth to develop such an elaborated interface as the macros are used by few people (as you say). I think there is nothing wrong with different commands for different types of options.